### PR TITLE
Build: Update browserstack settings

### PR DESCRIPTION
**Pull request by leobalter**
This update remove some browsers from the default set in order
to avoid timeout errors.

It also includes the Microsoft Edge browser.

Ref #815 

I'll also use this PR to check for possible timeouts.

### CODE REVIEW

File: browserstack-all.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		},
+		{
+			"os": "iOS",
+			"os_version": "8.3"
+		},
+		{
+			"os": "Windows",
+			"os_version": "10",
+			"browser": "edge",
+			"browser_version": "0.11"
```
Comment (mzgol) [inline browserstack-all.json]: I wouldn't specify this if not needed; Edge build numbers may change quite frequently.

File: browserstack-all.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+			"os": "iOS",
+			"os_version": "7.0"
+		},
+		{
+			"os": "iOS",
+			"os_version": "8.3"
+		},
+		{
+			"os": "Windows",
+			"os_version": "10",
```
Comment (mzgol) [inline browserstack-all.json]: Edge implies Windows 10 so this shouldn't be needed either. Can't you just write `edge_current`?

File: browserstack.json [hunk 3]
```diff
@@ -29,16 +27,10 @@
...
-			"os": "iOS",
-			"os_version": "7.0"
-		},
-		{
-			"os": "iOS",
-			"os_version": "8.3"
+			"os": "Windows",
+			"os_version": "10",
+			"browser": "edge",
+			"browser_version": "0.11"
```
Comment (mzgol) [inline browserstack.json]: See above remarks

File: browserstack-all.json (hunk 2 above)
Comment (leobalter) [inline browserstack-all.json]: There isn't an alias for Edge yet.

Setting it with the alias or omitting one of these properties will make browserstack to break and close silently.

Comment (mzgol) [inline browserstack-all.json]: Ah, I see.

**Comment by leobalter** [comment]
Just a note: this pr doesn't update browserstack-runner to 0.3.4 anymore as I've got some issues with it. I'll need to reflect this on the commit message

**Comment by jzaefferer** [comment]
@leobalter afaik BrowserStack fixed some issue with Edge in the API, can you update this to address @mzgol's comments?

**Comment by jzaefferer** [comment]
Also 0.3.5 is out and works fine on my machine. Worth a try here as well.

**Comment by leobalter** [comment]
trying browserstack 0.3.6

**Comment by leobalter** [comment]
@jzaefferer browserstack-runner is breaking on the newer versions. I'll isolate this problem to another issue.

**Comment by leobalter** [comment]
@mzgol I was able to use edge_latest, but `edge_current` and just `edge` breaks browserstack-runner (from 0.3.0 to 0.3.6)

**Pull request closed without merging**